res/layout/list_fragment.xml

```diff
@@ -81,4 +85,52 @@
             </ScrollView>
     </android.support.v4.widget.SwipeRefreshLayout>
 
-</FrameLayout>
\ No newline at end of file
+</FrameLayout>
+    <com.getbase.floatingactionbutton.FloatingActionsMenu
+        android:id="@+id/fab_main"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentEnd="true"
+        fab:fab_addButtonColorNormal="@color/owncloud_blue_accent"
```

Comment:
Yes you are right, ```color_accent``` would be the right one, unfortunately not part of the ```setup.xml``` at the moment so it couldn't be customized :(
I'll change it to color_accent, open issue then remains to make this available for branding. --> https://github.com/owncloud/android/commit/6c83af4521c5566d353e3ac281689679ad35b24f

using primary_button_background_color would enable branding though, but wouldn't be the right "name". So we could use it so branding still works for the next release plus I would really like to rewrite setup and color XMLs to have appropriate color definitions in setup.xml and then to the internally used names in the color.xml thus everything could be branded easily.

If this is in line with your strategy I would open a new issue where we can have a discussion on how to proceed with this and I can get a better understanding about how the setup.xml is used and after that rework the color definition :)